src/components/Feed.js

```diff
@@ -0,0 +1,24 @@
+import { h } from 'preact'
+import { useEffect, useState } from 'preact/hooks'
+import { isTrendingArticlesGroup } from 'utils'
+import { ListView } from 'components'
+import { useI18n } from 'hooks'
+import { getTrendingArticles } from 'api'
+
+export const Feed = ({ isExpanded, lang, containerRef }) => {
+  const [trendingArticles, setTrendingArticles] = useState([])
+  const i18n = useI18n()
+
+  useEffect(() => {
+    const [request, abort] = getTrendingArticles(lang)
+    request.then(setTrendingArticles)
+    return abort
+  })
+
+  return (
+    <div class={`feed ${isExpanded ? 'expanded' : 'collapsed'}`}>
+      {!isExpanded && <div class='cue' />}
+      {isTrendingArticlesGroup() && <ListView items={trendingArticles} header={i18n('feed-header')} containerRef={containerRef} />}
```

Comment:
I don't know if `isTrendingArticlesGroup()` is needed here. This is already checked in the Search component and if the Feed component is included it means the user must be in the trending group.

Reply:
You're right, this will be needed here only when the topics feed will be used